Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM golang:1.15-alpine AS dev
+# syntax=docker/dockerfile:1.2
+
+FROM --platform=$BUILDPLATFORM golang:1.21-alpine AS dev
```

Comment:
As we're also bumping our version of Golang, can we update go.mod to make the minimum supported be 1.19?